### ISSUE TITLE
Ender 3 v2

### ISSUE THREAD

**Issue by maticekkkk** [Jyers/Marlin#2089]
Hello to all.

I have updated my printer to Jyers and now when i use bed leveling it crash to the frame.. like is on the video..  and that happend on both sides left or right. What can i do to fix this ?

Another questuion...

I got sprite extruder pro kit 300.. How to change that on firmware  to get 300 celcius?

Thanks for help.

https://github.com/Jyers/Marlin/assets/85075278/d166cac6-d6c1-4458-b3d4-ce3c53c2cdf7

**Comment by russkoon** [comment]
Same here only I'm using the original hot-end pls CR Touch.

**Comment by maticekkkk** [comment]
Will try this now.
https://youtu.be/pxBGBjLC2rk
Meybe will work.

V sre., 26. jul. 2023, 02:56 je oseba russkoon ***@***.***>
napisala:

**Comment by maticekkkk** [comment]
Yeah it works fine now i have install that firmware i post youtune url. And
this settings if u have cr touch

V sre., 26. jul. 2023, 02:56 je oseba russkoon ***@***.***>
napisala:
